src/current/cockroachcloud/folders.md

```diff
@@ -0,0 +1,360 @@
+---
+title: Organize CockroachDB Cloud Clusters Using Folders
+summary: Learn how to use folders to organize CockroachDB Cloud clusters and manage access to them.
+toc: true
+docs_area: manage
+---
+
+{% include_cached feature-phases/limited-access.md %}
+
+This page explains how to organize and manage access to your {{ site.data.products.db }} organization's clusters using folders. For more details about managing access to {{ site.data.products.db }} resources, refer to [Managing Users, Roles, and Service Accounts in {{ site.data.products.db }}](managing-access.html).
+
+## How folders work
+
+By default, clusters are created at the level of the organization. Folders provide additional ways to organize clusters and manage access to them. A folder can contain both clusters and descendent folders.
+
+Each folder that you create has an ID. To create a cluster in a folder or to move a cluster into a folder, you set its `parent_id` field to the folder's ID. A folder can contain a mix of child folders and clusters.
```

Comment:
The last sentence here is a dupe from the previous paragraph, so maybe keep one of them

Reply:
Done